MessageService/src/main/java/com/beaconfire/messageservice/security/JwtFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.beaconfire.messageservice.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Optional;
+
+//The jwt filter that we want to add to the chain of filters of Spring Security
+@Component
+public class JwtFilter extends OncePerRequestFilter {
+
+    private JwtProvider jwtProvider;
+
+    @Autowired
+    public void setJwtProvider(JwtProvider jwtProvider) {
+        this.jwtProvider = jwtProvider;
+    }
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String path = request.getRequestURI();
+        System.out.println(path);
+//        if (path.equals("/user-service/user/authenticate") || path.equals("/user-service/user/create")) {
```

Comment:
Remove these debugging message if not needed

Reply:
removed